### ISSUE TITLE
react/sort-prop-types: conflict between callbacksLast and requiredFirst

### ISSUE THREAD

**Issue by smollweide** [namics/eslint-config-namics#2]
example:
```
AppNavDrawer.propTypes = {
	docked: PropTypes.bool.isRequired,
	style: PropTypes.object,
	onRequestChangeNavDrawer: PropTypes.func.isRequired
};
```

current config:
```
'react/sort-prop-types': [2, {
	ignoreCase: true,
	callbacksLast: true,
	requiredFirst: true
}],
```

**Comment by smollweide** [comment]
Will be solved with `eslint-plugin-react` version 6.2.1

**Comment by smollweide** [comment]
Done: [v2.0.0](https://github.com/namics/eslint-config-namics/releases/tag/2.0.0)
